### ISSUE TITLE
Investigate automated accessibility testing (WCAG) using with AXE js library on browserstack

### ISSUE THREAD

**Issue by petterwildhagen** [equinor/energyvision#306]
Investigate automated accessibility testing (WCAG) using with AXE js library on browserstack

**Comment by petterwildhagen** [comment]
Using https://www.cdnpkg.com/axe-core/file/axe.min.js/

**Comment by nilsml** [comment]
Should we conclude this task @petterwildhagen?
Then we can set up tasks for what is needed to be done, at least for now that we are in development mode:
1. Configure automated a11y testing on CI
2. Set up reporting to Slack
3. Add documentation to wiki

**Comment by nilsml** [comment]
Do you have an update on this work @petterwildhagen?
We should probably create some new tasks for the actual work and close this.
Let's talk about that next week.

**Comment by petterwildhagen** [comment]
Sven Van Steenis  9:06 AM
@pewi here for example: https://github.com/equinor/energyvision/blob/main/.github/workflows/web.yaml#L43
we call the test script that is in the package.json at web/package.json
I believe the run: command in the github actions workflow file is just a way of running commands in a terminal
I haven't used it myself but you should also be able to say which kind of shell you want the script to run in, with bash probably being the default
for example:
jobs:
  job1:
    runs-on: ubuntu-latest
    defaults:
      run:
        shell: bash
        working-directory: scripts
